Events/MessageEvent.cs

```diff
@@ -106,8 +106,14 @@ static async Task DeleteAndWarnAsync(DiscordMessage message, string reason, Disc
         
         static async Task DeleteAndWarnAsync(MockDiscordMessage message, string reason, DiscordClient client, bool wasAutoModBlock = false)
         {
+            var channel = message.Channel;
+            
             if (!wasAutoModBlock)
                 _ = message.DeleteAsync();
+            else
+                if (channel.Type is DiscordChannelType.GuildForum)
+                    channel = await client.GetChannelAsync(Program.cfgjson.ForumChannelAutoWarnFallbackChannel);
```

Comment:
Is it possible to fetch this once and reference that instance later? We used to have issues where D#+ wouldn't correctly cache `GetChannelAsync` calls and then we'd get rate limited if there's a lot of instances in a short space of time (e.g. raid spam)

Reply:
Sure—I could throw it into `Program.cs` and fetch on startup, if that's alright

Reply:
Yeah that makes sense